lib/LedgerSMB/Scripts/payment.pm

```diff
@@ -1406,7 +1406,7 @@ my %amount_to_be_used;
 my %ovp_repeated_invoices;
 my %invoice_id_amount_to_pay;
 my $count;
-my $warning = $Payment->{"warning"};
+my $warning = $Payment->{'warning'};
```

Comment:
below, you decided to leave `warning` unquoted. I agree with that choice, but then this one should be unquoted as well -- for consistency.